### PR TITLE
🌱 flavorgen: bump kubevip to v0.6.4 and import from file

### PR DESCRIPTION
**Pull request by chrischdi**
**What this PR does / why we need it**:

* Bumps kube-vip to v0.6.4
* Adjusts flavorgen to import kube-vip.yaml from file and add required changes on top.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: templates/cluster-template-ignition.yaml [hunk 1]
```diff
@@ -89,30 +89,42 @@ spec:
...
-            - name: svc_election
-              value: "true"
-            image: ghcr.io/kube-vip/kube-vip:v0.6.3
-            imagePullPolicy: IfNotPresent
+            - name: address
+              value: ${CONTROL_PLANE_ENDPOINT_IP}
+            - name: prometheus_server
+              value: :2112
+            image: ghcr.io/kube-vip/kube-vip:v0.6.4
+            imagePullPolicy: Always
```
Comment (fabriziopandini) [inline templates/cluster-template-ignition.yaml:127]: Should this be IfNotPresent so it works with pre-loaded images?

**Review by fabriziopandini — COMMENTED** [review]
nice, just one nit from my side

**Comment by sbueringer** [comment]
@chrischdi PTAL. Maybe a first flake from the new claim IP code

**Comment by chrischdi** [comment]
> @chrischdi PTAL. Maybe a first flake from the new claim IP code

xref fix: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/2663

**Pull request merged**